handlers/lookup_test.go

```diff
@@ -92,23 +92,26 @@ var _ = Describe("Lookup", func() {
 			reg.LookupReturns(pool)
 		})
 
-		It("sends a bad request metric", func() {
-			Expect(rep.CaptureBadRequestCallCount()).To(Equal(1))
+		It("does not send a bad request metric", func() {
+			Expect(rep.CaptureBadRequestCallCount()).To(Equal(0))
 		})
 
-		It("Sets X-Cf-RouterError to unknown_route", func() {
-			Expect(resp.Header().Get("X-Cf-RouterError")).To(Equal("unknown_route"))
+		It("Sets X-Cf-RouterError to no_endpoints", func() {
+			Expect(resp.Header().Get("X-Cf-RouterError")).To(Equal("no_endpoints"))
 		})
 
-		It("returns a 404 NotFound and does not call next", func() {
+		It("returns a 503 NotFound and does not call next", func() {
```

Comment:
You change the status code but not the message associated with the code.

Reply:
Resolved in https://github.com/cloudfoundry/gorouter/pull/251/commits/8c0eb34791893c96976965c37237241d639c4c49